crates/fuel-streams-core/src/types.rs

```diff
@@ -45,6 +47,18 @@ pub enum TransactionKind {
     Upload,
 }
 
+impl From<&Transaction> for TransactionKind {
```

Comment:
This is now already [implemented here](https://github.com/FuelLabs/data-systems/blob/main/crates/fuel-streams-core/src/transactions/types.rs)

Reply:
used